src/support_sphere/lib/presentation/components/auth/login_form.dart

```diff
@@ -142,7 +157,7 @@ class _LoginButton extends StatelessWidget {
         return state.status.isInProgress
             ? const CircularProgressIndicator()
             : ElevatedButton(
-                onPressed: state.isValid
+                onPressed: context.read<LoginCubit>().state.isLoginButtonEnabled
```

Comment:
```suggestion
                onPressed: state.isLoginButtonEnabled
```
SInce this is under a `BlocBuilder` you have access to the state directly with the `state` input variable

Reply:
updated!